swagger2.0/oic.wk.scenemember.swagger.json

```diff
@@ -0,0 +1,220 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "OCF Scene Member",
+    "version": "v1.1.0-20161107",
+    "license": {
+      "name": "OCF Data Model License",
+      "url": "https://openconnectivityfoundation.github.io/core/LICENSE.md",
+      "x-copyright": "Copyright 2016-2019 Open Connectivity Foundation, Inc. All rights reserved."
+    },
+    "termsOfService": "https://openconnectivityfoundation.github.io/core/DISCLAIMER.md"
+  },
+  "schemes": [
+    "http"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/SceneMemberResURI" : {
+      "get": {
+        "description": "Single Link that models a scene member.\n",
+        "parameters": [
+          {
+            "$ref": "#/parameters/interface-baseline"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description" : "",
+            "x-example": {
+              "rt": ["oic.wk.scenemember"],
+              "id": "0685B960-FFFF-46F7-BEC0-9E6234671ADC1",
+              "n": "my binary switch (for light bulb) mappings",
+              "link": {
+                "href": "binarySwitch",
+                "rt":   ["oic.r.switch.binary"],
+                "if":   ["oic.if.a", "oic.if.baseline"],
+                "eps": [
+                  {"ep": "coap://[fe80::b1d6]:1111", "pri": 2},
+                  {"ep": "coaps://[fe80::b1d6]:1122"},
+                  {"ep": "coap+tcp://[2001:db8:a::123]:2222", "pri": 3}
+                ]
+              },
+              "SceneMappings": [
+                {
+                  "scene":           "off",
+                  "memberProperty":  "value",
+                  "memberValue":     "true"
+                },
+                {
+                  "scene":           "Reading",
+                  "memberProperty":  "value",
+                  "memberValue":     "false"
+                },
+                {
+                  "scene":           "TVWatching",
+                  "memberProperty":  "value",
+                  "memberValue":     "true"
+                }
+              ]
+            },
+            "schema": {
+              "$ref": "#/definitions/SceneMember"
+            }
+          }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "interface-baseline" : {
+      "in" : "query",
+      "name" : "if",
+      "type" : "string",
+      "enum" : ["oic.if.baseline"]
+    },
+    "interface-all" : {
```

Comment:
Since both "interface-baseline" and "interface-all" contain the same elements, can't we remove "interface-all" since it's not referenced anywhere?